packages/service-library/requirements/_base.in

```diff
@@ -22,6 +22,8 @@ sqlalchemy
 psycopg2-binary
 pyyaml
 
+# used for monitoring of slow callbacks
+pyinstrument
```

Comment:
Since you add a new dependency **in a library**, you should compile the requirements of the services it includes
1. search which services are affected by
```cmd
grep --include=\*.in -nrw  -e 'packages/service-library/requirements/_base.in'
```
2. compile upgrade each service
```cmd
cd services/service/requirements
make touch
make reqs upgrade=pyinstrument
```

Reply:
Nice one, will do this.

Reply:
Ran this on all packages and no other packages require extra dependencies.
There are only some dependencies which are trying to be updated, like pytest-coverage etc... Those are out of the scope of this PR, will ignore them.

Reply:
did you use ``upgrade= pyinstrument`` option??

Reply:
I always forget about that option, usually I end up checking stuff manually.

Reply:
@GitHK Please do not forget to upgrade the requirements of the services! 
Right now it works because the servicelib ``setup`` is installing pyinstrument but the frozen requirements do not show all the dependencies